c2corg_ui/static/js/apiservice.js

```diff
@@ -579,6 +579,41 @@ app.Api.prototype.createImages = function(files, document) {
   return promise;
 };
 
+/**
+ * @return {!angular.$q.Promise<!angular.$http.Response>}
+ */
+app.Api.prototype.readForum = function() {
+  var alerts = this.alerts_;
+
+  var promise = this.getDiscourseLastest();
+  promise.catch(function(response) {
+    alerts.addError(response);
+  });
+  return promise;
+};
```

Comment:
Do you really need 2 distinct methods for querying the Discourse API?
I would recommend to:
- either have only one methode
- or create a generic (private) discourse GET method that would be used by the actual public method that will get the latest topics.